Description: Fix a typo in the error message for unreadble config file.
Author: Simon Chopin <chopin.simon@gmail.com>
Origin: vendor
Forwarded: not-needed
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/alot/settings/utils.py
+++ b/alot/settings/utils.py
@@ -25,7 +25,7 @@
         config = ConfigObj(infile=configpath, configspec=specpath,
                            file_error=True, encoding='UTF8')
     except (ConfigObjError, IOError):
-        raise ConfigError('Couls not read %s' % configpath)
+        raise ConfigError('Could not read %s' % configpath)
     except UnboundLocalError:
         # this works around a bug in configobj
         msg = '%s is malformed. Check for sections without parents..'
